ckanext/harvest/harvesters/base.py

```diff
@@ -178,6 +178,8 @@ def _create_or_update_package(self, package_dict, harvest_object):
 
                 else:
                     log.info('Package with GUID %s not updated, skipping...' % harvest_object.guid)
+                    harvest_object.report_status = 'not modified'
```

Comment:
I'd much rather report_status was only updated in queue.py, rather than introduce a new place where it potentially gets updated. Also, this is a deprecated method to be updating.

My preference would be for fetch_stage() to return True, False or "unchanged" to queue.py and set report_status there.